test/fixture.h

```diff
@@ -15,10 +15,46 @@
 // GraftServerTestBase fixture
 //
 //It has:
-//1. class MainServer based on Looper
+//1. class MainServer based on GraftServer
 //2. class Client based on mongoose
 //3. internal class TempCryptoNodeServer to simulate upstream behavior
 
+namespace detail
+{
+
+class GSTest : public graft::GraftServer
+{
+public:
+    GSTest(graft::Router& httpRouter, bool ignoreInitConfig)
+        : m_httpRouter(httpRouter)
+        , m_ignoreInitConfig(ignoreInitConfig)
+    { }
+    bool ready() const { return graft::GraftServer::ready(); }
+    void stop() { graft::GraftServer::stop(); }
+    graft::GlobalContextMap& getContext() const { assert(m_looper); return m_looper->getGcm(); }
+    graft::Looper& getLooper() const { assert(m_looper); return *m_looper.get(); }
+    virtual ~GSTest() { }
+    GSTest(const GSTest&) = delete;
+    GSTest& operator = (const GSTest&) = delete;
```

Comment:
I think it's good idea to put all special for class functions (ctor + dtor) at the beginning and together. Do not mix it with public methods. It could be general rule for our team.